index.html

```diff
@@ -23,15 +23,100 @@
       </ul>
-    </section>
+    </section>    
   </div>
+  <section class="grid-container" id="works">
+    <div class="cards" id="card-1">
+      <div class="snapshoot-placeholders" id="snapshoot-placeholder1"><img src="images/img-project1.png" alt="Project-1 Snapshoot"></div>
+      <div class="info-blocks" id="info-block1">
+        <h1 class="project-titles" id="project-title1">Tonic</h1>
+        <h2 class="project-infos" id="info-project1">CANOPY <span class="gray">&bull; Back End Dev &bull; 2015</span></h2>
+        <p class="project-descriptions" id="desc-project1">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="tag-lists">
+          <li class="tags">html</li>
+          <li class="tags">css</li>
+          <li class="tags">javascripts</li>
+        </ul>
+        <button class="btn" type="button">See Project</button>
+      </div>
+    </div>
+    <div class="cards" id="card-2">
+      <div class="snapshoot-placeholders" id="snapshoot-placeholder2"><img src="images/img-project2.png" alt="Project-2 Snapshoot"></div>
+      <div class="info-blocks" id="info-block2">
+        <h1 class="project-titles" id="project-title2">Multi-Post Stories</h1>
+        <h2 class="project-infos" id="info-project2">CANOPY <span class="gray">&bull; Back End Dev &bull; 2015</span></h2>
+        <p class="project-descriptions" id="desc-project2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="tag-lists">
+          <li class="tags">html</li>
+          <li class="tags">css</li>
+          <li class="tags">javascripts</li>
+        </ul>
+        <button class="btn" type="button">See Project</button>
+      </div>
+    </div>
+    <div class="cards" id="card-3">
+      <div class="snapshoot-placeholders" id="snapshoot-placeholder3"><img src="images/img-project3.png" alt="Project-3 Snapshoot"></div>
+      <div class="info-blocks" id="info-block3">
+        <h1 class="project-titles" id="project-title3">Tonic</h1>
+        <h2 class="project-infos" id="info-project3">CANOPY <span class="gray">&bull; Back End Dev &bull; 2015</span></h2>
+        <p class="project-descriptions" id="desc-project3">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="tag-lists">
+          <li class="tags">html</li>
+          <li class="tags">css</li>
+          <li class="tags">javascripts</li>
+        </ul>
+        <button class="btn" type="button">See Project</button>
+      </div>
+    </div>
+    <div class="cards" id="card-4">
+      <div class="snapshoot-placeholders" id="snapshoot-placeholder4"><img src="images/img-project4.png" alt="Project-4 Snapshoot"></div>
+      <div class="info-blocks" id="info-block4">
+        <h1 class="project-titles" id="project-title4">Multi-Post Stories</h1>
```

Comment:
- [ ] `h1` is only used once for the title of the page, so kindly use another `hx` tag here.